Added requirements (#1)
krishna63/crudApp: README.md

# crudApp
I am learning from scratch how to build an application with react using crud , routing and environment set up

I am going to add my TOdO for joining into IBM

TODO:
I should be familiar with github

Learnt on 09/sep
I have learnt to fork the project
I have learnt to clone the project using command git clone <url>
Next i have to create a branch using the command 

>     git checkout -b <branchname> <copyfrom the branch>

Author: Please add a period at the end of the sentence

# crudApp
I am learning from scratch how to build an application with react using crud , routing and environment set up

I am going to add my TOdO for joining into IBM

TODO:
I should be familiar with github

Learnt on 09/sep
I have learnt to fork the project
I have learnt to clone the project using command git clone <url>
Next i have to create a branch using the command.
    git checkout -b <branchname> <copyfrom the branch>
    ex: git checkout -b addingPause  master
List of commands available in git link: https://git-scm.com/doc

Commands:-------------------------------------------
git clone <url> ---> for cloning to the local directory

> git checout -b addingPauseReq master ----> creating a branch in my repository

Reviewer: ```suggestion
git checkout -b addingPauseReq master ----> creating a branch in my repository and it is taking `master` as a reference to create a branch
```

git status --->>> to see the added and updated files info

> git add . / git Add Readme ---> trying to add a file or code or folder 

Reviewer: ```suggestion
git add . / git Add Readme ---> This command will add all the unstaged files to staged. If we want to add only a particular file then we have to use as follows
git add <name of the folder/name of the file>
```

git commit -m "message"   ----->>> commiting 

> git push origin addingPauseReq     ---->>> It will push the code tp branch addingPauseReq

Reviewer: ```suggestion
git push origin addingPauseReq     ---->>> It will push the code to branch addingPauseReq
```

> git reset --head  ---->>> will save the previous files.

Reviewer: ```suggestion
git reset  --hard -->>> will reset all the unstaged files. Meaning you changed few files then the changed files will be in unstaged state, and you decided no need of the changes you have made so you can use the above command to revert all of the unstaged files
```

# crudApp
I am learning from scratch how to build an application with react using crud , routing and environment set up

I am going to add my TOdO for joining into IBM

TODO:
I should be familiar with github

Learnt on 09/sep
I have learnt to fork the project
I have learnt to clone the project using command git clone <url>
Next i have to create a branch using the command.
    git checkout -b <branchname> <copyfrom the branch>
    ex: git checkout -b addingPause  master
List of commands available in git link: https://git-scm.com/doc

Commands:-------------------------------------------
git clone <url> ---> for cloning to the local directory
git checkout -b addingPauseReq master ----> creating a branch in my repository and it is taking `master` as a reference to create a branch
git status --->>> to see the added and updated files info
git add . / git add Readme ---> trying to add a file or code or folder 
git commit -m "message"   ----->>> commiting 
git push origin addingPauseReq     ---->>> It will push the code tp branch addingPauseReq
git reset --hard  ---->>> will save the previous files.

> git status

Author: ```suggestion
git status --->> will show the list of status files 
```

Author: @krishna63 : The above command is self explanatory, i dont think we require further explanation
